### PR TITLE
i18n(zh-cn): Optimize Chinese translation

### PR DESCRIPTION
**Pull request by liruifengv**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

- Optimize Chinese translation
- Update the landing page title. Use `<br/>` to wrap.
- Update the landing page tagline

689f920cd55862968599eead0dd6fea

### CODE REVIEW

File: docs/src/content/docs/zh/index.mdx
Comment (delucis) [inline docs/src/content/docs/zh/index.mdx:3]: Is the repeated “Astro 构建 Astro 构建” here intentional?

**Review by delucis — COMMENTED** [review]
Thanks for this polish @liruifengv! Love the new title ✨ 

Just want to double check a detail but otherwise LGTM!

File: docs/src/content/docs/zh/index.mdx
Comment (liruifengv) [reply docs/src/content/docs/zh/index.mdx:3]: Sorry, it was my mistake, we should delete the repeat

Comment (delucis) [reply docs/src/content/docs/zh/index.mdx:3]: Thanks for the confirmation! I’ll let you make the change before I merge:
```suggestion
description: Starlight 帮助你使用 Astro 构建漂亮、高性能的文档网站。
```

**Review by delucis — APPROVED** [review]
Thanks @liruifengv!

**Pull request merged**
